katalog/keycloak/sm.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2022 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: keycloak-metrics
+  labels:
+    app: keycloak
```

Comment:
There are already 2 other svcs with this label, I would change that to `keycloak-metrics` to ease the service discovery done through the Service Monitor.

Reply:
Good point!
What about 2 labels, like `app: keycloak` and `component: metrics`?
Without the namespace I think the filter `app=keycloak` could be useful 😄

Reply:
I think it's fine, my point was to have a unique way to address the service associated to the metrics. 👍